util/build_configs/cray-internal/process-template.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+"""
+Replace substitutable strings starting with // in templates.
+"""
+
+import argparse, sys
+from string import Template
+
+def create_parser():
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument('--template', type=argparse.FileType('r'),
+                        help='The input template',
+                        default=sys.stdin)
+    parser.add_argument('--output', type=argparse.FileType('w'),
+                        help='The output file with substitutions made',
+                        default=sys.stdout)
+    return parser
+
+def cli_to_dict(keys):
+    substitutions = {}
+    for arg in keys:
+        words = arg.split('=', 1)
+        substitutions[words[0]] = words[1]
+    return substitutions
+
+# Use // as the delimiter
```

Comment:
Update template name (`SlashTemplate` -> `AtTemplate`, or `MyTemplate` or something) and change (or just remove the comment above )